playground/src/containers/app.tsx

```diff
@@ -431,9 +431,12 @@ export class App extends React.Component<AppProps, AppState>
         const activeWorkspace = this.state.workspaces[this.state.activeWorkspaceName];
         const contents = workspaceToShareContent(activeWorkspace, this.inputEditors);
         const data = await share(contents);
+
         if (data) {
             window.history.pushState({}, 'CUE Playground', '?id=' + data + window.location.hash);
             this.setState({ ...this.state, saved: true, showSaveURL: true });
+
+            updateRobotsMetaNoIndex(new URLSearchParams(window.location.search), 'id');
```

Comment:
This is now adding the meta tag after it's been shared to the current page. When a person clicks on a shared link nothing will be added because they don't share it themselves: they just open the link. so if we want to do something like this we need to do it on page load. Also I'm not sure if this will work at all since it probably executes after the robots have already done their thing but it's worth a try.. But we do need something on load: not on share.